scripts/translate.js

```diff
@@ -80,9 +80,19 @@ async function main() {
       __dirname,
       `../src/Translation/locales/${locale}.json`
     )
-    const currentTranslations = require(localePath)
+    const currentTranslations = flat(require(localePath))
+    // remove obsolete keys
+    const translations = await translation.fetch(DEFAULT_LOCALE)
```

Comment:
I think this is already available as `mainTranslations`? If not, I think it should be fetched outside the for loop, because it shouldn't change

Reply:
Moreover, can't we perform this process on the main loop above? where `const translations = await translation.fetch(locale)`

The script is a bit hard to follow to me (not you'r code, what's already there), so if you can't or it's too hard don't worry about it

Reply:
I also got a little bit confused when reading this code. This is what I could do without to much refactoring: 50dfb4a